api/src/main/java/org/pmiops/workbench/firecloud/FireCloudConfig.java

```diff
@@ -23,7 +23,6 @@
 public class FireCloudConfig {
 
   public static final String X_APP_ID_HEADER = "X-App-ID";
```

Comment:
Can you add a note describing our use of this header, and what expectations exist on the FireCloud side?

An alternative place to put this documentation / comment would be as a comment on the WorkbenchConfig class definition. I know we have zero comments there right now, but that's a pretty poor state of affairs & it's never too late to start leaving breadcrumbs of knowledge :)

Reply:
👍 to this, adding it into WorkbenchConfig, and adding a tech debt ticket to do an audit of our config file and figure out what isn't clear so we can comment it.

Reply:
Filed: https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=30&projectKey=RW&view=planning&selectedIssue=RW-2616&epics=visible&selectedEpic=RW-1801